typos.toml

```diff
@@ -0,0 +1,43 @@
+# https://github.com/crate-ci/typos/blob/master/docs/reference.md
+
+[files]
+extend-exclude = [
+  "*.csv",
+  "*.eps",
+  "*.lvt",
```

Comment:
Maybe a bit brute to ignore all the lvt files. But otherwise we end with a not-too-short list of lvt file paths to ignore.